README.md

```diff
@@ -4,50 +4,71 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Theodo-UK/sls-dev-tools/blob/master/LICENSE)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/Theodo-UK/sls-dev-tools)
 [![Follow](https://img.shields.io/twitter/follow/SlsDevTools?style=social)](https://twitter.com/SlsDevTools)
+![Maintained](https://img.shields.io/maintenance/yes/2020.svg)
+[![Tweet](./img/tweetBadge.svg)](https://twitter.com/intent/tweet?text=@SlsDevTools)
```

Comment:
![image](https://user-images.githubusercontent.com/38322167/79959009-7c54d680-847b-11ea-9e8d-7a60ec90a8e0.png)
Some badge not showing and looks lik its removd the gitter badge?